library/src/main/java/com/novoda/downloadmanager/lib/CollatedDownloadInfo.java

```diff
@@ -0,0 +1,40 @@
+package com.novoda.downloadmanager.lib;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class CollatedDownloadInfo {
+
+    private final long totalSize;
+
+    static CollatedDownloadInfo collateInfo(Map<Long, DownloadInfo> mDownloads, DownloadInfo info) {
```

Comment:
param `mDownloads` along with `info` 

To `m` or to do not `m`, that's the question :dancer: